packages/theme/getters/productGetters.ts

```diff
@@ -199,15 +198,37 @@ export const getFormattedPrice = (price: number) => {
   }).format(price);
 };
 
-export const getBreadcrumbs = (product: ProductInterface, category?: Category): AgnosticBreadcrumb[] => {
+const getCategoryBreadcrumbs = (category: CategoryInterface): AgnosticBreadcrumb[] => {
+  let breadcrumbs = [];
```

Comment:
```suggestion
  let breadcrumbs : AgnosticBreadcrumb[] = [];
```
You can just do that instead of using `as AgnosticBreadcrumb` below